modules/readme/Makefile

```diff
@@ -29,7 +32,5 @@ readme/lint:
 ## Create README.md by building it from README.yaml
 readme/build: $(README_DEPS)
 	@gomplate --file $(README_TEMPLATE_FILE) \
-		--out $(README_FILE) \
-		--datasource config=$(README_YAML) \
-		--datasource includes='file://$(shell pwd)/?type=text/plain'
-	@echo "Generated $(README_FILE) from $(README_TEMPLATE_FILE) using data from $(README_TEMPLATE_YAML)"
+		--out $(README_FILE) 
+	@echo "Generated $(README_FILE) from $(README_TEMPLATE_FILE) using data from $(USED_DATASOURCE)"
```

Comment:
What's wrong with using `README_TEMPLATE_YAML`? I don't understand why we need a new variable.

Reply:
due possible overriding, it is not necessary?